.github/workflows/Operator-CI.yaml

```diff
@@ -20,6 +21,12 @@ jobs:
     steps:
       - name: Checkout
```

Comment:
Please adjust the checkout to use merge branch and add recheckout as done in https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/.github/workflows/Manager-CI.yaml#L48-L51